27.md

```diff
@@ -1,22 +1,64 @@
 NIP-27
 ======
 
-Text Note References
---------------------
+Nostr Unassuming Markup Language (NUML)
+---------------------------------------
 
-`draft` `optional`
+`draft` `mandatory`
 
-This document standardizes the treatment given by clients of inline references of other events and profiles inside the `.content` of any event that has readable text in its `.content` (such as kinds 1 and 30023).
+Unless explicitly specified otherwise, text content in nostr events MUST follow the guidelines below for balancing human readability and embedded links. Client MAY parse and display content to improve the aesthetics of displayed content.
 
-When creating an event, clients should include mentions to other profiles and to other events in the middle of the `.content` using [NIP-21](21.md) codes, such as `nostr:nprofile1qqsw3dy8cpu...6x2argwghx6egsqstvg`.
+# Recognized Formatting
 
-Including [NIP-18](18.md)'s quote tags (`["q", "<event-id> or <event-address>", "<relay-url>", "<pubkey-if-a-regular-event>"]`) for each reference is optional, clients should do it whenever they want the profile being mentioned to be notified of the mention, or when they want the referenced event to recognize their mention as a reply.
+- A `>` at the beginning of a line followed by a single space MAY be used to indicate quotations
+- A `#` at the beginning of a word MAY be used to indicate a topic and SHOULD have a corresponding `t` tag
```

Comment:
I wonder if we should clarify that a "topic" is a hashtag and not a chapter or section as it is on Markdown.